identity/webapp/src/frontend/Registration/AccountValidated.test.tsx

```diff
@@ -35,29 +35,19 @@ describe('AccountValidated', () => {
     ).toBeInTheDocument();
   });
 
-  it('displays a success message', () => {
-    renderPage(
-      '/validated?message=Your%20email%20was%20verified.%20You%20can%20continue%20using%20the%20application.&success=true'
-    );
-    expect(screen.getByRole('alert')).toHaveTextContent(
-      'Your email was verified. You can continue using the application.'
-    );
-  });
-
   it('displays a failure message', () => {
     renderPage(
       '/validated?message=This%20URL%20can%20be%20used%20only%20once&success=false'
     );
-    expect(screen.getByRole('alert')).toHaveTextContent(
-      'This URL can be used only once'
-    );
+    expect(screen.getByText('This URL can be used only once')).toBeTruthy();
   });
 
   it('shows a link to login on success', () => {
     renderPage('/validated?success=true');
-    const link = screen.getByRole('link');
+    const links = screen.getAllByRole('link');
+    const link = links[1];
```

Comment:
Why is this `link[1]` - it feels quite brittle, myaybe we could get it by text rather?

Reply:
Yeah, I wondered about that. I think the approach would be a bit more involved (get by text, then establish that the enclosing element was a link with a particular href), but probably a better way to do it.

Reply:
While we're here - should these tests exist in `playwright` rather than it's own style of testing?